applications/callflow/src/module/cf_voicemail.erl

```diff
@@ -1629,6 +1676,13 @@ should_require_pin(MailboxJObj) ->
         'false' -> kzd_voicemail_box:pin_required(MailboxJObj)
     end.
 
+-spec allow_ff_rw(kz_json:object()) -> boolean().
+allow_ff_rw(MailboxJObj) ->
+    case ?DEFAULT_ALLOW_FF_RW of
+        'true' -> 'true';
+        'false' -> kzd_voicemail_box:pin_required(MailboxJObj)
```

Comment:
Why does 'pin_required' matter for whether ff/rewind is allowed?

Reply:
@jamesaimonetti latest code has been updated for this, please take a look :)